### PR TITLE
add Code & Markdown Editor Component

### PR DESCRIPTION
**Pull request by sFLASHs**

### CODE REVIEW

**Comment by kyrylo-burmelov** [comment]
[Code & Markdown Editor Component](https://trello.com/c/ObpyddFn/14-code-markdown-editor-component)

File: client/src/components/pages/create-task/common/create-tabs/create-code/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import AceEditor from 'react-ace';
+import { Ace } from 'ace-builds';
```
Comment (taras-dubyk) [inline client/src/components/pages/create-task/common/create-tabs/create-code/index.tsx:3]: combine imports from ace-builds

File: client/src/components/pages/create-task/common/create-tabs/create-code/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import 'ace-builds/src-noconflict/mode-javascript';
+import 'ace-builds/src-noconflict/theme-tomorrow';
+import 'ace-builds/src-noconflict/ext-language_tools';
+import 'ace-builds/src-noconflict/ext-beautify';
+
+config.set('basePath', 'https://cdn.jsdelivr.net/npm/ace-builds@1.4.8/src-noconflict/');
+config.setModuleUrl(
+	'ace/mode/javascript_worker',
+	'https://cdn.jsdelivr.net/npm/ace-builds@1.4.8/src-noconflict/worker-javascript.js',
+);
```
Comment (taras-dubyk) [inline client/src/components/pages/create-task/common/create-tabs/create-code/index.tsx:17]: Seems redundant. Try to import ace-builds before AceEditor.

File: client/src/components/pages/create-task/common/create-tabs/create-tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	const handleChangeTab = (tabNumber: number) => {
+		setTab(tabNumber);
+	};
+
+	const handleFullScreen = () => {
+		setFullScreen((state) => !state);
+	};
+
+	return (
+		<div className={`${styles.root} ${fullScreen ? styles['edit__fullscreen'] : styles['edit']}`}>
```
Comment (taras-dubyk) [inline client/src/components/pages/create-task/common/create-tabs/create-tabs.tsx:23]: Use [clsx](https://www.npmjs.com/package/clsx)

File: client/src/components/pages/create-task/common/create-tabs/header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Icon } from '@blueprintjs/core';
+import ReactTooltip from 'react-tooltip';
+import { IconTaskPageTab } from 'common';
+import { TCreateTabs } from '../types';
+
+import styles from './styles.module.scss';
+
+interface IProps {
```
Comment (taras-dubyk) [inline client/src/components/pages/create-task/common/create-tabs/header/index.tsx:9]: Rename to IHeaderProps

File: client/src/components/pages/create-task/common/create-tabs/create-text/index.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { ITabProps } from '../types';
+
+import styles from './styles.module.scss';
+
+export const CreateText: React.FC<ITabProps> = ({ text, onChange, editable }) => {
+	const [value, setValue] = useState<string>(text);
```
Comment (taras-dubyk) [inline client/src/components/pages/create-task/common/create-tabs/create-text/index.tsx:7]: We don't need local state for tab value here. Value should be recieved from parent container.

**Review by taras-dubyk — CHANGES_REQUESTED** [review]
Issues should be resolved.

**Review by taras-dubyk — APPROVED** [review]

**Pull request merged**